api/src/feeds/impl/feeds_api_impl.py

```diff
@@ -76,7 +76,7 @@ def get_feed(self, id: str, db_session: Session) -> BasicFeed:
             .filter(Feed.data_type != "gbfs")  # Filter out GBFS feeds
             .filter(
                 or_(
-                    Feed.operational_status == None,  # noqa: E711
+                    Feed.operational_status == "published",
                     Feed.operational_status != "wip",
```

Comment:
Please remove the `!=wip` conditional the equality to "published" give the correct result.(The same in other similar queries)
```suggestion
                    Feed.operational_status == "published",
```